Catappult/appcoins-wallet-android
# Eskl 19/user input

**What does this PR do?**

- Moves userName input to 2048
- Removes productlabel from URI

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

With 2048 from eskl-19/user-input, check if flow is working as expected. You should now see userInput on 2048 side (before wallet intent)

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-19](https://aptoide.atlassian.net/browse/ESKL-19)

**Questions:**

N/A



**Code Review Checklist**
